src/style.css

```
@@ -0,0 +1,135 @@
+html {
+  font-family: Roboto, sans-serif;
+  --color-main-text: #060b37;
+  --color-secondary-text: #616070;
+  --color-accent: #00acdc;
+}
+
+body {
+  margin: 0;
+}
+
+h2 {
+  margin: 0;
+}
+
+p {
+  margin: 0;
+}
```

You can combine these selectors
```suggestion
body,
h2,
p {
  margin: 0;
}
```